src/ui/profile_preferences.ui

```diff
@@ -55,6 +55,19 @@
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_19">
```

Comment:
oops, missed this just now.  Its not hard to move the IRC group back to a tab though. :stuck_out_tongue: